native/mod_manager/mod_manager.c

```diff
@@ -1206,6 +1206,36 @@ static void mark_node_removed(nodeinfo_t *node)
         node->mess.num_remove_check = 0;
     }
 }
+static const proxy_worker_shared *read_shared_by_node(request_rec *r, nodeinfo_t *node)
+{
+    void *sconf = r->server->module_config;
+    int i, port;
+    char *name = apr_pstrcat(r->pool, "balancer://", node->mess.balancer, NULL);
+    if (sscanf(node->mess.Port, "%u", &port) != 1) {
+        return NULL; /* something is wrong */
+    }
+    proxy_server_conf *conf = (proxy_server_conf *)ap_get_module_config(sconf, &proxy_module);
+    proxy_balancer *balancer = (proxy_balancer *)conf->balancers->elts;
```

Comment:
This must be at the beginning of the function in order httpd with maintainer mode to pass.

Reply:
fixed.